cheroot/test/test_ssl.py

```diff
@@ -30,6 +30,7 @@
     _get_conn_data,
     _probe_ipv6_sock,
 )
+from ..wsgi import Gateway_10
```

Comment:
I feel uneasy about this change. What is the meaning of this change? Can you confirm this change doesn't affect other tests in this module?

Reply:
That's exactly why I postponed this review... While it looks like the author knows what they're doing, it's more complicated than it looks.

Reply:
sorry, I was lazy in adding tests... while it shouldn't have effected them, I've split it off into a separate fixture so it doesn't introduce scope into the existing tests